atom/common/native_mate_converters/callback.cc

```diff
@@ -132,6 +116,22 @@ v8::Local<v8::Value> CreateFunctionFromTranslater(
                           v8::Object::New(isolate));
 }
 
+// func.bind(func, arg1).
```

Comment:
does this have to move? It makes the change less clear

Reply:
@bridiver the move pulls it out of the unnamed `namespace`. It's used here https://github.com/brave/muon/pull/507/files/5c42c5d9755fef62518ae8a883e6ffb9aa8c20c4#diff-9e5169c80bbdb5c416b1d94cea1cf2ebR80

Reply:
I see, ok